io-engine/src/bdev/nx.rs

```diff
@@ -0,0 +1,128 @@
+//! Allows creation of a nexus through a URI specification rather than gRPC or
+//! direct function call. This is not intended for the usual product operation
+//! but for testing and benchmarking.
```

Comment:
Could you please add some URI examples for this nexus:// namespace with some details on URI arguments ?

Reply:
Sure done, ptal